### PR TITLE
Anonymous users going to /admin on a shib-only app get shown an empty form

### PR DESCRIPTION
**Pull request by simong**
Example:

Steps to reproduce:
- Disable local auth
- Enable shibboleth auth
- Sign out on the app
- Go to /admin

### CODE REVIEW

**Comment by simong** [comment]
I've attached a patch that shows the "Sign in with Raven" button on /admin if Shib auth is enabled

File: apps/timetable/admin/ui/js/index.js [hunk 1]
```diff
@@ -72,14 +72,19 @@ define(['gh.core', 'gh.constants', 'moment', 'moment-timezone', 'gh.header', 'gh
      * @private
      */
     var renderLoginForm = function() {
-        $('#gh-subheader, #gh-content-description').height(350);
+        if (gh.config.enableLocalAuth && !gh.config.enableShibbolethAuth) {
+            $('#gh-subheader, #gh-content-description').height(350);
+        } else {
+            $('#gh-subheader, #gh-content-description').height(150);
+        }
+
```
Comment (Coenego) [inline apps/timetable/admin/ui/js/index.js]: Maybe it's better to do:

```
var height = 150;
if (gh.config.enableLocalAuth && !gh.config.enableShibbolethAuth) {
    height = 350;
} 
$('#gh-subheader, #gh-content-description').height(height);
```

**Comment by Coenego** [comment]
One minor comment, back to @simong for follow-up

**Comment by simong** [comment]
Done, pushed follow-up

**Comment by simong** [comment]
Would you mind doing a 1.0.1 release for this? You don't have to generate any artefacts, just tag it. I can publish to S3

**Pull request merged**